velox/functions/prestosql/aggregates/ApproxPercentileAggregate.cpp

```diff
@@ -371,10 +371,11 @@ class ApproxPercentileAggregate : public exec::Aggregate {
 
         auto value = decodedValue_.valueAt<T>(row);
         auto weight = decodedWeight_.valueAt<int64_t>(row);
-        VELOX_USER_CHECK_GE(
-            weight,
-            1,
-            "The value of the weight parameter must be greater than or equal to 1.");
+        VELOX_USER_CHECK(
```

Comment:
Does this check also happen on the java side ? Should the error messages be consistent ?

Reply:
The implementation used on Java side is QDigest and has a different limit.  I can try to enlarge this one to 2^63-1 though.

Reply:
Set it to 2^63-1 will likely overflow when we retrieve the quantiles.  Let's keep that one for now since it should be large enough.